Sources/_CryptoExtras/RSA/RSA.swift

```diff
@@ -318,6 +319,65 @@ extension _RSA.Signing {
     }
 }
 
+extension _RSA.Encryption {
+    public typealias PublicKey = _RSA.Signing.PublicKey
+    public typealias PrivateKey = _RSA.Signing.PrivateKey
+}
+
+extension _RSA.Encryption {
+    public struct RSAEncryptedData: ContiguousBytes {
```

Comment:
Any reason not to just use `Data` directly here?

Reply:
I was mimicking the design of `RSASignature` - if memory serves, I literally copy-pasted and changed the name.

Reply:
Yes, but RSA signatures are nominally at least somewhat structured. Encrypted bytes aren't.